examples/scripts/drive_cycle.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 import os
 
-os.chdir(pybamm.__path__[0] + "/..")
+os.chdir(pybamm.__path__[0] + "/.." + "/..")
```

Comment:
Also, could you check if this line is needed anymore? If not, we should remove it unless we have inclinations to keep it to historical reasons :)

```suggestion
```

Reply:
yes we can